docs/getting-started/generated-app.md

```diff
@@ -0,0 +1,53 @@
+---
+id: generated-app
+title: The generated app
+sidebar_label: The Generated App
+slug: /generated-app
+---
+
+# The Generated App
+
+Applications created with Amplication are generated as human-readable and editable TypeScript code.
+
+In this article you will find the details on the structure of the application and the technologies behind it.
+
+## Technologies
+
+The generated app is built with the following amazing open-source technologies:
+
+### Server-side
+
+- [NestJS](https://nestjs.com/) - A progressive Node.js framework for building efficient, reliable and scalable server-side applications.
+- [Prisma](https://www.prisma.io/) - Next-generation ORM
```

Comment:
@yuval-hazaz after two marketing fluff comments, let's think how you list these components without inserting the marketing fluff from their website descriptions. Do you need to provide a description? If there's a link, developers who aren't familiar with these packages will jump over there and read about them. Bekitzur, I'd remove all the descriptions and just list the items they need to install.